index.html

```diff
@@ -1,21 +1,70 @@
-<html>
-<head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
-</head>
-<body>
-<div class="container">
-    <header></header>
-    <section class="section-1">
-        <h1>I'm Marine Glad to see you</h1>
-        <p>I'm a software Developer! I can help you build aproduct, 
-            feature or website look through some of my work and 
-            exprience! if you like what you se & have aproject you need, 
-            don't hestiate to contact me</p>
-    </section>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <header>
+          <nav>
+            <div class="logo">
+              <a>
```

Comment:
- [ ] Great job adding an `<a>` tag as suggested by the previous reviewers, please add the `href=#` attribute to the `<a>` tag 😄 
- [ ] The `<nav>` tag should only enclose the hamburger icon for good semantics 
- [ ] Please the hamburger icon should be an image and not a span element